tox.ini

```diff
@@ -0,0 +1,32 @@
+[tox]
+envlist = coverage-clean, py27, py33, py34, py35, coverage-report
+
+[testenv]
+commands =
+    coverage run setup.py ftest
+setenv =
+  COVERAGE_FILE=.coverage.{envname}
+deps =
+    {toxinidir}/zope.app.publication-4.0.0a1.dev0.tar.gz
+    coverage
+    zope.testrunner
+    ; ZEO pin to get compatibility for Python 3.3
+    ZEO <5.0
```

Comment:
Let's not pin the version now -- when there's a ZEO on PyPI that breaks one of our tox targets, we can pin it to an earlier version of that particular target (or maybe decide that version of Python doesn't need to be supported any more).